### PR TITLE
OTT-1146: Added code to update vast unwrap status in the prometheus stats

### PR DESCRIPTION
**Pull request by Pubmatic-Supriya-Patil**
# Description

Please add change description or link to ticket, docs, etc.

# Checklist:

- [ ] PR commit list is unique (rebase/pull with the origin branch to keep master clean).
- [ ] JIRA number is added in the PR title and the commit message.
- [ ] Updated the `header-bidding` repo with appropiate commit id.
- [ ] Documented the new changes.

For Prebid upgrade, refer: https://inside.pubmatic.com:8443/confluence/display/Products/Prebid-server+upgrade

### CODE REVIEW

**Review by pm-jaydeep-mohite — COMMENTED** [review]
Update PR title

File: modules/pubmatic/vastunwrap/unwrap_service.go [hunk 1]
```diff
@@ -15,44 +15,34 @@ import (
 func doUnwrap(m VastUnwrapModule, bid *adapters.TypedBid, userAgent string, unwrapURL string, accountID string, bidder string) {
 
 	startTime := time.Now()
+	var respStatus string
 	defer func() {
-		respTime := time.Since(startTime)
-		m.MetricsEngine.RecordRequestTime(accountID, bidder, respTime)
+		// respTime := time.Since(startTime)
+		// m.MetricsEngine.RecordRequestTime(accountID, bidder, respTime)
```
Comment (pm-jaydeep-mohite) [inline modules/pubmatic/vastunwrap/unwrap_service.go:21]: Remove commented lines

Comment (Pubmatic-Supriya-Patil) [reply modules/pubmatic/vastunwrap/unwrap_service.go:21]: As discussed with viral we can have this code with comments. If required I can remove it

**Review by pm-jaydeep-mohite — APPROVED** [review]
LGTM

**Pull request merged**